prov/usnic/src/usdf_ep_dgram.c

```diff
@@ -420,6 +420,7 @@ static const struct fi_domain_attr dgram_dflt_domain_attr = {
 	.resource_mgmt = FI_RM_DISABLED,
 	.mr_mode = OFI_MR_BASIC_MAP | FI_MR_LOCAL,
 	.cntr_cnt = USDF_DGRAM_CNTR_CNT
+	.mr_iov_limit = 1
```

Comment:
I think the `mr_iov_limit` is only relevant for `fi_mr_regv` and `fi_mr_regattr`. Since we don't currently support those operations, I think we should be returning 0 instead of 1. 

@shefty Does that sound right?